### DISCUSSION TITLE
i have chatterino nightly but dont see the new twitch animated emotes?

### DISCUSSION THREAD

**Discussion by benjiPKS** [Chatterino/chatterino2#2898] [Q&A] [answered]
how exactly do i turn the setting on? ive tried every keyword i can think of but im not the brightest if you havent figured that out yet LULW

**Comment by apa420** [comment]
Did you download the 7tv version or just chatterino.
If you downloaded 7tv post an issue there instead.

**Reply by benjiPKS** [reply]
i just copied some of the files from 7tv into my nightly peepoG

**Reply by benjiPKS** [reply]
feelsdankman i figured it out. do answered pweaes

**Reply by Mm2PL** [reply]
Please actually provide the answer to this, so if others find this they can use it too. :)

**Reply by benjiPKS** [reply]
All you have to do is copy the 7tv version of chatterino nightly and paste it into your chatterino folder and make sure to replace the files. Since I'm an idiot I skipped it LULW

**Comment by zneix** [comment, marked as answer]
Case is solved, see https://github.com/SevenTV/chatterino7/discussions/16
